pkg/vault/contracts/CompositeLiquidityRouterHooks.sol

```diff
@@ -36,7 +36,11 @@ abstract contract CompositeLiquidityRouterHooks is BatchRouterCommon {
         ERC4626
     }
 
-    // Used to keep track of input tokens that have already been processed in nested pools.
+    // Used to keep track of input tokens that have already been processed in nested pools. This is distinct from
+    // `_currentSwapTokensIn`, which keeps track of the input tokens in a batch swap (e.g., preventing
```

Comment:
hmm maybe we can say in a batch swap / add liquidity operation?
This is actually the CLR and there's no batch swap in here, so that might sound a bit confusing.